dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/message/MessageImpl.java

```diff
@@ -297,5 +652,1699 @@ void setContextualProperty(String key, Object v) {
+                httpBasePath = NOT_FOUND;
+            } else if (key == ASYNC_POST_RESPONSE_DISPATCH) {
+                ret = asyncPostDispatch;
+                asyncPostDispatch = NOT_FOUND;
+            } else if (key == SECURITY_CONTEXT) {
+                ret = securityContext;
+                securityContext = NOT_FOUND;
+            } else if (key == AUTHORIZATION_POLICY) {
+                ret = authorizationPolicy;
+                authorizationPolicy = NOT_FOUND;
+            } else if (key == CERT_CONSTRAINTS) {
+                ret = certConstraints;
+                certConstraints = NOT_FOUND;
+            } else if (key == AbstractHTTPDestination.SERVICE_REDIRECTION) {
+                ret = serviceRedirection;
+                serviceRedirection = NOT_FOUND;
+            } else if (key == HTTP_SERVLET_RESPONSE) {
+                ret = httpServletResponse;
+                httpServletResponse = NOT_FOUND;
+            } else if (key == RESOURCE_METHOD) {
+                ret = resourceMethod;
+                resourceMethod = NOT_FOUND;
+            } else if (key == ONE_WAY_REQUEST) {
+                ret = oneWayRequest;
+                oneWayRequest = NOT_FOUND;
+            } else if (key == ASYNC_RESPONSE) {
+                ret = asyncResponse;
+                asyncResponse = NOT_FOUND;
+            } else if (key == THREAD_CONTEXT_SWITCHED) {
+                ret = threadContextSwitched;
+                threadContextSwitched = NOT_FOUND;
+            } else if (key == OutgoingChainInterceptor.CACHE_INPUT_PROPERTY) {
+                ret = cacheInputProperty;
+                cacheInputProperty = NOT_FOUND;
+            } else if (key == PhaseInterceptorChain.PREVIOUS_MESSAGE) {
+                ret = previousMessage;
+                previousMessage = NOT_FOUND;
+            } else if (key == AbstractHTTPDestination.RESPONSE_HEADERS_COPIED) {
+                ret = responseHeadersCopied;
+                responseHeadersCopied = NOT_FOUND;
+            } else if (key == SSE_EVENT_SINK) {
+                ret = sseEventSink;
+                sseEventSink = NOT_FOUND;
+            } else if (key == REQUESTOR_ROLE) {
+                ret = requestorRole;
+                requestorRole = NOT_FOUND;
+            } else if (key == PARTIAL_RESPONSE_MESSAGE) {
+                ret = partialResponse;
+                partialResponse = NOT_FOUND;
+            } else if (key == EMPTY_PARTIAL_RESPONSE_MESSAGE) {
+                ret = emptyPartialResponse;
+                emptyPartialResponse = NOT_FOUND;
+            } else if (key == ENDPOINT_ADDRESS) {
+                ret = endpointAddress;
+                endpointAddress = NOT_FOUND;
+            } else if (key == INBOUND_MESSAGE) {
+                ret = inboundMessage;
+                inboundMessage = NOT_FOUND;
+            }
+
+            return ret == NOT_FOUND ? null : ret;
+        }
+        return super.remove(key);
+    }
+    
+    @Override
```

Comment:
This isn't needed in the sub class.  Its existence in the super class is sufficient

Reply:
This get method is still there.